pennylane/math/utils.py

```diff
@@ -193,6 +194,8 @@ def requires_grad(tensor):
 
     Args:
         tensor (tensor_like): input tensor
+        interface (str): The name of the interface. If not provided,
+            will be determined automatically.

```

Comment:
```suggestion
        interface (str): The name of the interface. Will be determined automatically 
        if not provided.
```

Splitting hairs here :)

Reply:
Yours reads better!